integration-libs/opf/components/opf-checkout-payment-wrapper/opf-checkout-payment-wrapper.component.ts

```diff
@@ -56,10 +58,8 @@ export class OpfCheckoutPaymentWrapperComponent implements OnInit {
           configurationId: String(this.selectedPaymentId),
           cartId: this.activeCartId,
           // TODO: Move below as a part of a whole OPF configuration?
-          resultURL:
-            'http://localhost:4200/electronics-spa/en/USD/checkout/review-order',
-          cancelURL:
-            'http://localhost:4200/electronics-spa/en/USD/checkout/review-order',
+          resultURL: `http://localhost:4200/${this.config.opf?.successUrl}`,
```

Comment:
hard coded domain for demo purpose, i haven't investigate the best way to get client URL domain: routingService? window obj? config?

Reply:
I used a OpfUrlHandlerService shared between components that is picking the current domain.
having the absolute path already in config would involve fetching the current domain before any components load, I am not sure how to do it.

Reply:
as discussed, it is now handle by service using router.
```
 resultURL: this.opfUrlHandlerService.getAsoluteUrl(
              'paymentVerificationResult'
            ),
            cancelURL: this.opfUrlHandlerService.getAsoluteUrl(
              'paymentVerificationCancel'
```
            ),